book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] rentals should also have the getter and setter accessibility. You can just include it in the attr_accessor.